### ISSUE TITLE
[Request] Begin packaging Orange as a Flatpak on Linux

### ISSUE THREAD

**Issue by heliophane** [biolab/orange3-installers#53]
Orange would be great to have as a flatpak, and would make installing it and running it much easier that it is right now.

**Comment by markotoplak** [comment]
I agree. But i do not think this is currently the priority for anyone in the group.

If anyone would like to attempt it, we'd be grateful.

**Comment by heliophane** [comment]
I'll take a look into doing it myself, and can dedicate some time to it this weekend. I can't make any promises, and I don;t have experience but I'l try.

I currently use Orange for a university class and would love to be able to make it easier for future students as well.

**Comment by heliophane** [comment]
@markotoplak I started a repo and began work on the manifest file. I will continue to work throughout the weekend

https://github.com/heliophane/orange-flatpak

**Comment by theoden8** [comment]
Hi @markotoplak, I have a working attempt here https://github.com/theoden8/orange3-flatpak
also see https://github.com/biolab/orange3/issues/5403
CC @chrislee35

**Comment by theoden8** [comment]
It's practically ready, may I use the official author (biolab) name in the metadata and put down orangedatamining.com site for verfiication?
See the above PR for contents
